### PR TITLE
[TMOB-1209]: Name releases (title) more clearly on GitHub

### PR DESCRIPTION
**Pull request by neodym**
## Description
This PR will rename the following release type titles:

| before | after |
| - | - |
| App Store Release Candidate | Beta |
| App Store Release | Release |
| App Store Hotfix Release | Release (hotfix) |

## Ticket
https://wetransfer.atlassian.net/browse/TMOB-1209

## Conversation about this topic
https://wetransfer.slack.com/archives/C3ZV6K8K0/p1655728927908339

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/91472abc-6f61-4fdd-813c-094cfa7e5dbf" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>WeTransferPRLinterTests: Executed 18 tests (0 failed, 0 retried, 0 skipped) in 13.852 seconds</td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
WeTransferPRLinter | 92.19% | ✅
WeTransferPRLinter | 0.00% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 9205bb95b02363a0c08c6093a8f82c540e22f005
</p>

File: Fastlane/deployment_lanes.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
   if is_changed_since_last_tag == false
     tag_name = create_tag_name(xcodeproj: xcodeproj, target: target)
-    cancel_message = 'A new Release is cancelled as there are no changes since the last available tag.'
+    cancel_message = 'A new Beta build has been cancelled as there are no changes since the last available tag.'
```
Comment (BasThomas) [inline Fastlane/deployment_lanes.rb:35]: I think it'd be nice to (also) keep a reference to "release candidate" here; what do you think?

**Review by BasThomas — APPROVED** [review]

File: Fastlane/deployment_lanes.rb (hunk 1 above)
Comment (neodym) [reply Fastlane/deployment_lanes.rb:35]: Well the inspiration for this change was this slack conversation:
https://wetransfer.slack.com/archives/C3ZV6K8K0/p1655728927908339

The idea was to have a clear difference between what's "just" internal and what's going to be delivered to the customers. Therefore it was wished to not mention "releases" too much :)

File: Fastlane/deployment_lanes.rb [hunk 2]
```diff
@@ -74,7 +74,7 @@
 
   # Create a new GitHub release
   last_non_candidate_tag = latest_github_non_candidate_tag
-  release_title = "#{tag_name} - App Store Release Candidate"
+  release_title = "#{tag_name} - Beta"
```
[the remark this answers is not in the document]
Comment (neodym) [reply Fastlane/deployment_lanes.rb:77]: same answer above ;)

File: Fastlane/deployment_lanes.rb [hunk 3]
```diff
@@ -105,7 +105,7 @@
     UI.important "TestFlight delivery failed because a build is already in review, but continuing anyway!"
   end
 
-  success_message = 'A new Release Candidate has been published.'
+  success_message = 'A new Beta has been published.'
```
[the remark this answers is not in the document]
Comment (neodym) [reply Fastlane/deployment_lanes.rb:108]: same answer above ;)

**Review by AvdLee — APPROVED** [review]

File: Fastlane/deployment_lanes.rb (hunk 1 above)
Comment (BasThomas) [reply Fastlane/deployment_lanes.rb:35]: Gotcha — just don't want it to become confusing for us. 😄

Comment (neodym) [reply Fastlane/deployment_lanes.rb:35]: Same here. The build lane is already called "beta". I think it'll be much clearer for us if less things are called "release" ;)

**Pull request merged**